src/main/java/uk/gov/companieshouse/api/accounts/repository/AccountsNoteRepositoryFactory.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.companieshouse.api.accounts.repository;
```

Comment:
Is there a test class for this ?

Reply:
Added test class for above mentioned class